### PR TITLE
Add shortnames for MachineConfig and MachineConfigPool CRDs

### PR DESCRIPTION
**Pull request by LorbusChris**
**- What I did**
Added mc and mcd shortnames for MachineConfig and MachineConfigPool CRDs respectively.

**- How to verify it**
`oc describe mc` and `oc describe mcp`

**- Description for the changelog**

Added mc and mcd shortnames for MachineConfig and MachineConfigPool CRDs respectively.

### CODE REVIEW

File: manifests/machineconfigpool.crd.yaml [hunk 1]
```diff
@@ -40,3 +40,7 @@ spec:
     singular: machineconfigpool
     # kind is normally the CamelCased singular type. Your resource manifests use this.
     kind: MachineConfigPool
+    # shortNames allow shorter string to match your resource on the CLI
+    shortNames:
+    - mcp
+
```
Comment (runcom) [inline manifests/machineconfigpool.crd.yaml]: drop empty lines

**Comment by runcom** [comment]
needs a `make update` as well (look at the verify job)

**Comment by kikisdeliveryservice** [comment]
Did we confirm that these are unique?  A little worried about `mc` since it's so short it might have name collision elsewhere?

**Comment by runcom** [comment]
> Did we confirm that these are unique? A little worried about `mc` since it's so short it might have name collision elsewhere?

Kubernetes should be taking care of conflicts when _registering_ names and short names for a CRD *(trying to find the code that does that which I've forgotten now)

**Comment by runcom** [comment]
> Kubernetes should be taking care of conflicts when _registering_ names and short names for a CRD *(trying to find the code that does that which I've forgotten now)

can't find function but looking at the kubernetes repo there are various examples of how shortnames are expanded and checked for conflicts, CI would also shed some light on this

**Comment by kikisdeliveryservice** [comment]
> can't find function but looking at the kubernetes repo there are various examples of how shortnames are expanded and checked for conflicts, CI would also shed some light on this

Gotcha. Just wanted to make sure there wasn't something obvious we were missing esp since mc is so short. :+1:

**Comment by runcom** [comment]
sign-off from @derekwaynecarr that this isn't disrupting anything because of _adding_ it

**Pull request merged**